MOBILE_API.xml

```diff
@@ -2187,9 +2188,10 @@
     
     <struct name="DisplayCapabilities" since="1.0">
         <description>Contains information about the display capabilities.</description>
-        <param name="displayType" type="DisplayType" mandatory="true" deprecated="true" since="5.0">
+        <param name="displayType" type="DisplayType" removed="true" since="6.0">
```

Comment:
`mandatory` field still needs to be present due to the RPC spec rules, although I believe it would inherently be `mandatory="false"` when removed is true